src/headless/main.cpp

```diff
@@ -109,15 +112,28 @@ static std::chrono::nanoseconds run_pathing_test(AStarPathing& pathing) {
 	return end_time - start_time;
 }
 
-static bool run_headless(Dataloader::path_vector_t const& roots, bool run_tests) {
+static bool run_headless(fs::path const& root, std::vector<std::string>& mods, bool run_tests) {
 	bool ret = true;
+	Dataloader::path_vector_t roots = { root };
+	Dataloader::path_vector_t replace_paths = {};
 
 	GameManager game_manager { []() {
 		Logger::info("State updated");
 	} };
 
+	Logger::info("===== Loading mod descriptors... =====");
+	ret &= game_manager.load_mod_descriptors(std::move(mods));
```

Comment:
std::move(mods) does literally nothing

```suggestion
	ret &= game_manager.load_mod_descriptors(mods);
```

Reply:
Fixed